components/Actions/Action/Action.js

```diff
@@ -1,14 +1,52 @@
-import styles from './Action.module.css';
-
-const Action = ({ copy, isContact, title }) => {
-	const Component = isContact ? 'a' : 'div';
-	const classNames = `${styles.tile} ${title ? styles.hasTitle : ''} ${isContact ? styles.isContact : ''}`;
-	return (
-		<Component className={classNames} {...(isContact ? { href: 'mailto:sustainability@random.studio' } : {})}>
-			{title && <h3 className={styles.title}>{title}</h3>}
-			<p className={styles.copy}>{copy}</p>
-		</Component>
-	);
-}
-
-export default Action;
\ No newline at end of file
+import styles from "./Action.module.css";
+import { useEffect, useState, createRef } from "react";
+import ReactMarkdown from "react-markdown";
+import { disableBodyScroll, clearAllBodyScrollLocks } from "body-scroll-lock";
+import Modal from "../../Modal/Modal";
+
+const Action = ({ copy, markdown, isContact, title }) => {
+  const modalRef = createRef();
+  const [isOpen, setIsOpen] = useState(false);
+
+  useEffect(() => {
+    if (modalRef.current && isOpen) {
```

Comment:
Think an early exit with `return undefined` is more legible. Makes it clearer we only need to run clear when hook has run `disableBodyScroll`